### PR TITLE
squad_client/core/models.py: Add attachment_urls property to TestRuns

### PR DESCRIPTION
**Pull request by katieworton**
Add a property `attachement_urls` to TestRuns that returns the download URLs for a given TestRun.

### CODE REVIEW

**Comment by chaws** [comment]
Yeah, it looks good. When you're writing tests, could you please write one retrieving the file using one of those urls?

**Comment by katieworton** [comment]
@chaws - I have been trying to draft a basic unit test that reads either the download_urls or the attachments, but having difficulty creating the attachments in fixtures.py to be accessed in test_models.py. When I try out the feature in a script it appears to work, just not in my unit test.

If you have any pointers for this issue, that would be great 🙌

**Comment by katieworton** [comment]
@chaws unit tests now added and appear to work correctly - let me know if there's anything else required here 👍

File: tests/test_models.py [hunk 1]
```diff
@@ -194,6 +196,26 @@ def test_testrun_status(self):
...
+        """Test that created attachment entries can be accessed via the attachment_urls TestRun property."""
+        attachment_urls = self.testrun_attachment.attachment_urls
+        self.assertTrue(attachment_urls is not None)
+        self.assertTrue(len(attachment_urls) == 2)
+
+        attachment_contents = []
+        expected_attachment_contents = [b'attachment file 1 content', b'attachment file 2 content']
+
+        for attachment_url in attachment_urls:
+            request = requests.get(attachment_url)
```
Comment (chaws) [inline tests/test_models.py:214]: I think we can create an `Attachment` class with a `read` function within so we could do things like:
 
```python
# core/models.py

class Attachment(SquadObject):
    def __init__(self, url):
        self.url = url

    def read(self):
        response = SquadApi.get(self.url)
        # check response.status_code here
        return response.contents

class TestRun(SquadObject):
...
    @attachments.setter
    def attachments(self, attachments):
        if attachments:
            self.__attachments__ = []
            for a in attachments:
                self.__attachments__.append(Attachment(a["download_url"]))
...

# tests/test_models.py

...

contents = []
for a in testrun.attachments:
  contents.append(a.read())

...

```

This way we still centralize all HTTP calls to SquadApi class, which already handle retries and cookies and auth tokens.

And I just realized now that attachments in SQUAD are public. Let's chat about that too tomorrow.

**Comment by chaws** [comment]
I merged it now, and will do those changes myself, then we can discuss in the meeting a way that you can create a new command squad for downloading attachments

**Pull request merged**